2024-06-12 01:45:44,450 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2024-06-12 01:45:44,450 - root - INFO - Firebase initialized successfully
2024-06-12 01:45:44,694 - root - DEBUG - Users loaded: [{'username': 'admin', 'password': 'password'}, {'username': 'user1', 'password': 'password1'}, {'username': 'user2', 'password': 'password2'}]
2024-06-12 01:45:52,477 - root - DEBUG - Attempting login with username: admin, password: password
2024-06-12 01:45:52,477 - root - INFO - Login successful
2024-06-12 01:45:52,504 - root - INFO - Setting up case numbers tab
2024-06-12 01:45:52,511 - root - INFO - Loading case table data
2024-06-12 01:45:52,517 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2024-06-12 01:45:52,520 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2024-06-12 01:45:52,820 - urllib3.connectionpool - DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
2024-06-12 01:45:52,824 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): polapordb-default-rtdb.asia-southeast1.firebasedatabase.app:443
2024-06-12 01:45:54,089 - urllib3.connectionpool - DEBUG - https://polapordb-default-rtdb.asia-southeast1.firebasedatabase.app:443 "GET /cases.json HTTP/1.1" 200 522
2024-06-12 01:45:54,090 - root - DEBUG - Cases retrieved: {'47000': {'comment': 'Комментарий', 'creation_date': '2024-05-30 20:42:35', 'customer': 'Заказчик', 'name': 'Имя'}, '47001': {'comment': '', 'creation_date': '2024-06-01 23:26:16', 'customer': 'customer', 'name': 'Igor'}, '47002': {'comment': '', 'creation_date': '2024-06-02 22:01:01', 'customer': 'customer', 'name': 'Igor'}, '47003': {'comment': '', 'creation_date': '2024-06-01 23:32:06', 'customer': 'customer', 'name': 'Igor'}, '47004': {'comment': '', 'creation_date': '2024-06-01 23:32:21', 'customer': 'СКФ', 'name': 'Антон'}}
2024-06-12 01:45:54,090 - root - INFO - Case table data loaded
2024-06-12 01:45:54,094 - root - ERROR - Error setting up case numbers tab: 'MainWindow' object has no attribute 'populate_customers'
2024-06-12 01:45:54,704 - urllib3.connectionpool - DEBUG - https://polapordb-default-rtdb.asia-southeast1.firebasedatabase.app:443 "GET /cases.json HTTP/1.1" 200 522
2024-06-12 01:45:55,121 - urllib3.connectionpool - DEBUG - https://polapordb-default-rtdb.asia-southeast1.firebasedatabase.app:443 "GET /cases.json HTTP/1.1" 200 522
2024-06-12 01:45:55,580 - urllib3.connectionpool - DEBUG - https://polapordb-default-rtdb.asia-southeast1.firebasedatabase.app:443 "GET /proformas.json HTTP/1.1" 200 249
2024-06-12 01:45:55,712 - root - INFO - Application started successfully
